tutorials/customizing-base-map-styles/index.md

```diff
@@ -0,0 +1,379 @@
+---
+title: Customizing base map styles
+description: If you are getting bored of looking at the same two default basemaps in Earth Engine, or need to use a specific visualization for functionality/convenience, this tutorial will show you how to edit all of its characteristics.
+author: TC25
+tags: Earth Engine, Base Map, UI, Widgets
+date_published: 2019-09-16
+---
+
+[Open In Code Editor](https://code.earthengine.google.com/283c86dd3189788452b5fb64174a1b43)
+
+The user is going to learn how to change the options of the map object to create new base maps
+
+## The default maps in Earth Engine
+
+Earth Engine's base maps are those in Google's Map API. The default options include:
+
+- roadmap, which displays the default road map view,
+- satellite, which displays Google Earth satellite images,
+- hybrid, which displays a mixture of normal and satellite views, and
+- terrain, which displays a physical map based on terrain information.
+
+## Changing the basic map style
+
+We can start by changing the style of the base map. One easy fix is to invert the lightness to get a darker background, like so:
+
+```javascript
+
+var baseChange = [
+  "stylers": [ {"invert_lightness": true} ]
+]
+Map.setOptions("baseChange",{"baseChange":baseChange})
+
+```
+The main styler options include:
+
+ - hue: indicates the basic color
+ - lightness:indicates percentage change in brightness of element
```

Comment:
Nit: "an element" or "elements", here and below.

Reply:
Done